### DISCUSSION TITLE
Question - do not want to receive pictures

### DISCUSSION THREAD

**Discussion by ROSAUrsaMajor** [sly9/VoyagerTelegramBot#38] [Q&A] [answered]
Hello guys!

First of all, I would like to thank you for this great project. I haven't quite reached my goal with the bot yet, but I will. I'm not particularly fit when it comes to programming, so here comes a question that is probably quite easy to answer... :-)

What did I do: downloaded your latest release for Windows, also the zip file, unpacked it and copied the config.yml.example, renamed it and filled it with data (API ID, etc.).

The bot now sends me messages, but also every single raw image as .jpg. I would like to turn this off, but I can't find the button. :-)

Another issue is that the imaging session statistics are not arriving at my bot? Where could I start looking there?

All the best,
Markus

**Comment by sly9** [comment, marked as answer]
Hey Markus,

Thanks for trying out our bot!

We once had a config flag, which sets a lower bound of exposure in seconds. Only exposures longer than that will receive thumbnails.
We also had a config flag, which does exactly the same thing as you mentioned -- it controls whether images will be sent to telegram or not.

We are debating whether we should bring this back -- or some of them back.. keep tuned.

For the other question, the bot will send image statistics if you run a dragscript with sequence(s) in it. Ad-hoc shots, and maybe ad-hoc sequence runs will not trigger a stats image.

Hope this answers your question,

Best,
Liuyi
